dashboard/app/views/home/_homepage.html.haml

```diff
@@ -8,6 +8,8 @@
 - homepage_data[:topCourse] = @top_course
 - homepage_data[:isEnglish] = @is_english
 - homepage_data[:includeDanceParty] = DCDO.get("dance_projects", false)
+- hoc_launch = DCDO.get("hoc_launch", nil)
+- homepage_data[:includeMCAquatic] = !hoc_launch.nil?
```

Comment:
just to be safe, i think we should check the the value of `hoc_launch` matches an acceptable value ('mc' or 'dance':
```
- homepage_data[:includeMCAquatic] = ['mc', 'dance'].include? hoc_launch
```

Reply:
```suggestion
- homepage_data[:includeMCAquatic] = ['mc', 'dance'].include? hoc_launch
```